### PR TITLE
ADEN-2714 Test third prefooter

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 50 issues:
* MINOR 50 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "MH3U:_Monsters" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L102) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "126608052" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L165) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Krux/Page_1" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L714) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"pos\":\"TOP_LEADERBOARD\"" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L320) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"hostpre\":" 7 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L342) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"lang\":\"en\"" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L315) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"loc\":\"top\"" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L319) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"s0\":\"gaming\"" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L370) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "FFFFFF" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L200) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest-pluto" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L104) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "TOP_RIGHT_BOXAD" 6 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L177) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "DirectGpt; RemnantGpt; Liftium" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L677) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "leagueoflegends" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L47) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest" 41 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L34) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "Wikia" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "adtest-pad" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L776) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "zh.pad" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L54) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_left.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L197) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "monsterhunter" 5 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L102) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "HUB_TOP_LEADERBOARD" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L84) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"pform\":[\"pc\"]" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L421) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"pv\":\"1\"" 7 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L349) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "gameshub" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L84) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"s2\":\"article\"" 6 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L310) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "wka.hub/_ent_hub//hub" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L88) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "yugioh" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L45) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"cat\":[" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L343) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/ProvidersChain" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L676) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "ru.elderscrolls" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L98) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "naruto" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L46) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"s0c\":[\"gaming\"]" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L372) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "CORP_TOP_LEADERBOARD" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"ar\":\"4:3\"" 7 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L344) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"sex\":[\"m\"]" 6 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L356) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "src/test/resources/adsResources/wikia_skin_right.png" 9 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L198) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "runescape" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L44) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "AAAAAA" 9 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L199) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"esrb\":\"teen\"" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L316) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"dmn\":\"wikiacom\"" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L311) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Interstitial/NotScalable" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L794) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "TOP_LEADERBOARD" 12 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L165) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"pos\":\"TOP_RIGHT_BOXAD\"" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L395) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "%D0%9A%D0%B2%D0%B5%D1%81%D1%82%D1%8B_%28Skyrim%29" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L98) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Interstitial" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L787) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"src\":\"gpt\"" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L321) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "elderscrolls" 3 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L68) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"ref\":\"direct\"" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L314) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"s0v\":\"games\"" 4 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L371) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "\"top\":\"1k\"" 8 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L350) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)
* MINOR [Define a constant instead of duplicating this literal "wikia" 7 times.](https://github.com/Wikia/selenium-tests/blob/b2b4e66a88305f01d3cdb948383d98b31bc0fe1b/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L80) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java [hunk 1]
```diff
@@ -864,4 +864,16 @@ private AdsDataProvider() {
...
+    return new Object[][]{
+        {
+            "adtest", "", true
+        },
+        {
+            "adtest", "SyntheticTests/Prefooters", false
+        }
+    };
+  }
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: remove extra line

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java [hunk 2]
```diff
@@ -864,4 +864,16 @@ private AdsDataProvider() {
...
   }
 
+  @DataProvider
+  public static Object[][] adsMiddlePrefooter() {
+    return new Object[][]{
+        {
+            "adtest", "", true
+        },
+        {
+            "adtest", "SyntheticTests/Prefooters", false
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: what do you think about moving each param to new line? that would increase readability

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 3]
```diff
@@ -74,6 +75,9 @@
   private WebElement incontentBoxad;
   @FindBy(css = LIFTIUM_IFRAME_SELECTOR)
   private List<WebElement> liftiumIframes;
+  @FindBy(css = MIDDLE_PREFOOTER_CSS_SELECTOR)
+  private WebElement middlePrefooter;
+
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: remove extra line

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    AdsBaseObject adsBaseObject = new AdsBaseObject(driver, testedPage);
+    adsBaseObject.scrollToFooter();
+
+    boolean middlePrefooterOnPage = adsBaseObject.isMiddlePrefooterOnPage();
+
+    Assertion.assertEquals(middlePrefooterOnPage, middlePrefooterEnabled,
+                           "Middle prefooter on page: " + middlePrefooterOnPage);
+
+    if (middlePrefooterEnabled) {
+      adsBaseObject.verifyMiddlePrefooterAdPresent();
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java]: I would suggest converting `verify` method to return some value into `Assertion` here

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java (hunk 2 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: :no_good:

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java (hunk 4 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java]: :+1: 
I thought about it as well but it requires more refactoring.
Let's start to use this convention after refactoring tests.

Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrefooters.java]: :+1:

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by drets** [comment]
Thanks :wink:

**Pull request merged**